sdk/python/kfp/compiler_cli_tests/test_data/xgboost_sample_pipeline.json

```diff
@@ -4,13 +4,20 @@
       "executorLabel": "exec-chicago-taxi-trips-dataset",
       "inputDefinitions": {
         "parameters": {
+          "format": {
+            "defaultValue": "csv",
+            "parameterType": "STRING"
+          },
```

Comment:
This is a behavior change. Previously we "hide" this to workaround Vertex Pipelines backend's lack of support for optional inputs. Can we test submitting this and see if that limitation is gone?

Reply:
Tried and worked on Optional inputs and xgboost examples.